src/main/java/io/jenkins/update_center/DirectoryTreeBuilder.java

```diff
@@ -157,21 +156,15 @@ protected void stage(MavenArtifact a, File dst) throws IOException {
             throw new IOException("Failed to create " + parentFile);
         }
 
-        ProcessBuilder pb = new ProcessBuilder();
         if (System.getProperty("os.name").toLowerCase(Locale.US).contains("windows")) {
             return;
         }
-        pb.command("ln", "-f", src.getAbsolutePath(), dst.getAbsolutePath());
-        Process p = pb.start();
+        Path newLink = Paths.get(dst.getAbsolutePath());
+        Path existingFile = Paths.get(src.getAbsolutePath());
         try {
-            if (p.waitFor() != 0) {
-                throw new IOException("'ln -f " + src.getAbsolutePath() + " " + dst.getAbsolutePath() +
-                        "' failed with code " + p.exitValue() + "\nError: " + IOUtils.toString(p.getErrorStream()) + "\nOutput: " + IOUtils.toString(p.getInputStream()));
-            } else {
-                LOGGER.log(Level.INFO, "Created new download file " + dst + " from " + src);
-            }
-        } catch (InterruptedException ex) {
-            LOGGER.log(Level.WARNING, "Interrupted creating " + dst + " from " + src, ex);
+            Files.createSymbolicLink(newLink, existingFile);
```

Comment:
This is wrong: `ln` first takes the destination file, then the link file argument. This switches the order around.